src/multi_vector_simulator/utils/__init__.py

```diff
@@ -179,9 +214,44 @@ def compare_input_parameters_with_reference(folder_path, ext=JSON_EXT):
     if len(extra_parameters) > 0:
         answer[EXTRA_PARAMETERS_KEY] = extra_parameters
```

Comment:
Where do we use `answer[EXTRA_PARAMETERS_KEY]`? Do we print somewhere that we have too many keys in our inputs?

Reply:
We use it for the test of the input_template, it should have no extra parameter nor any missing ones. I use it for debugging as well :)